src/content/docs/validator/external-chains/xrp-evm.mdx

```diff
@@ -0,0 +1,162 @@
+# XRPL EVM
+
+import { Callout } from "/src/components/callout";
+
+Setup your XRPL EVM Mainnet or Testnet node.
+
+## Prerequisites
+
+- MacOS or Ubuntu 18.04+
+- Build-essential packages
+- Golang 1.17+
+- [Official Documentation](https://docs.xrplevm.org/pages/operators/getting-started/system-requirements)
+
+
+| Resource        | Minimum Recommendation       | Notes                                   |
+|-----------------|------------------------------|-----------------------------------------|
+| Operating System | Linux (AMD64)               | A recent Linux distribution (e.g., Ubuntu 20.04 LTS, CentOS 8, Debian 11) is strongly recommended for stability, security, and compatibility. |
+| CPU             | 4 Physical Cores or More     | Multi-core processors provide enough parallel processing to efficiently handle validation, API requests, and block synchronization. Higher core counts can enhance performance under heavy load. |
+| Memory (RAM)    | 32GB                         | Sufficient memory ensures smooth operation during peak network load, handling large state DB operations, and responding to complex smart contract queries. More memory may further improve performance. |
+| Storage         | 500GB NVMe SSD               | A high-performance NVMe SSD provides fast random access and write speeds, which are essential for maintaining a responsive node database and quickly processing state updates. More storage capacity may be required for archive nodes or long-term data retention. |
+| Network          | 100Mbps                     | A stable 100Mbps connection or faster ensures timely propagation of blocks, efficient peer discovery, and responsive API endpoints. Lower bandwidth may lead to delayed synchronization or performance bottlenecks. |
+
+## Install
+
+To get started with the node setup.
+
+<Callout emoji="💡">
+  This setup has been tested with `exrpd` version `6.0.0`
+</Callout>
+
+1. Install the latest binary from the official XRPL EVM node [repo](https://github.com/xrplevm/node/releases)
+2. Move the binary to your preferred directory
+3. Make the binary executable
+
+```bash
+chmod +x exrpd
+```
+
+4. Confirm if `exrpd` successfully installed by running version command
+
+```bash
+ exrpd version
+```
+
+## Node Configuration
+
+1. Create the .exrpd directory in your user’s home directory:
+```bash
+mkdir -p ~/.exrpd
+```
+Inside it, you will eventually see two subfolders. The first is `~/.exrpd/config`. The second is `~/.exrpd/data`.
+
+2. Set the chain id
+```bash
+exrpd config chain-id exrp_1440002-1
+```
+
+3. Initialize the node
+Make sure to replace `<your_moniker>` with a short ASCII-only name for your node (e.g., my-node).
+```bash
+exrpd init <your_moniker> --chain-id exrp_1440002-1
```

Comment:
@benjamin852 can we clarify this is for devnet only? i'd make a tabbd view and show testnet/mainnet as TBD

Reply:
ya good idea will add that in